lib/compat/wordpress-5.9/class-gutenberg-rest-global-styles-controller.php

```diff
@@ -218,18 +218,46 @@ protected function prepare_item_for_database( $request ) {
 	public function prepare_item_for_response( $post, $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
 		$config                           = json_decode( $post->post_content, true );
 		$is_global_styles_user_theme_json = isset( $config['isGlobalStylesUserThemeJSON'] ) && true === $config['isGlobalStylesUserThemeJSON'];
-		$result                           = array(
-			'id'       => $post->ID,
-			'settings' => ! empty( $config['settings'] ) && $is_global_styles_user_theme_json ? $config['settings'] : new stdClass(),
-			'styles'   => ! empty( $config['styles'] ) && $is_global_styles_user_theme_json ? $config['styles'] : new stdClass(),
-			'title'    => array(
-				'raw'      => $post->post_title,
-				'rendered' => get_the_title( $post ),
-			),
-		);
-		$result                           = $this->add_additional_fields_to_object( $result, $request );
-		$response                         = rest_ensure_response( $result );
-		$links                            = $this->prepare_links( $post->id );
+
+		$fields = $this->get_fields_for_response( $request );
+
+		// Base fields for every post.
+		$data = array();
+
+		if ( rest_is_field_included( 'id', $fields ) ) {
+			$data['id'] = $post->ID;
+		}
+
+		if ( rest_is_field_included( 'title', $fields ) ) {
```

Comment:
Should we include both raw and rendered here as well?

Reply:
That is what is happening in the line below 😄